individual/gql_mutations.py

```diff
@@ -140,6 +152,15 @@ def _validate_mutation(cls, user, **data):
                 IndividualConfig.gql_individual_delete_perms):
             raise ValidationError("mutation.authentication_required")
 
+        villages_qs = Location.objects.filter(individual__id__in=data['ids'], type='V')
```

Comment:
not sure what you are doing here filtering Location with individual_id (for id you need only one _)

Reply:
Good spotting. Fixed.